css/css-values/percentage-rem-low.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>REM value with :root set to low percentage</title>
+<link rel="author" title="Jon Ege Ronnenberg" href="mailto:jon.ronnenberg+wpt@gmail.com">
+<link rel="help" href="https://www.w3.org/TR/css-values-3/#rem">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#length-percentage-size-value">
+<link rel="match" href="../reference/ref-filled-green-100px-square-only.html">
```

Comment:
Note that I just picked this reference so that there would be *some* reference and this is interpreted as a reftest. You don't have to use a green square as your ref if that isn't the most useful.

If *just* the mismatch suffices to cover this test, that's even better.

Reply:
Well, we'd rather just a single match than a mismatch. I think the green square is reasonable. So I'd say drop the mismatch? If it matches that it inherently won't match the mismatch given here.

Reply:
I only did it because the docs says:

>Add another reference document which describes what the test should not look like using rel=mismatch
https://web-platform-tests.org/writing-tests/reftest-tutorial.html#more-practice

Should I use the _reference_ ref or use the ref from the first commit? A green square is perfectly fine as ref.

Reply:
We should clarify the docs here! That's definitely not the intention. The green square is the better option here, but testing a whole range with a testharness JS test would probably be better still.

Reply:
I will look into testharness.js based on Trevor Burnham's test in another PR